github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml

```diff
@@ -57,3 +57,34 @@ postsubmits:
           resources:
             requests:
               memory: "8Gi"
+
+    - name: publish-fedora-realtime-container-image
+      annotations:
+        testgrid-create-test-group: "false"
+      always_run: false
+      run_if_changed: "cluster-provisioning/images/vm-image-builder/fedora-realtime/.*"
+      decorate: true
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-kubevirtci-quay-credential: "true"        
+      cluster: prow-workloads
+      spec:
+        containers:
+          - image: quay.io/kubevirtci/golang:v20210316-d295087
```

Comment:
Why have you selected the golang image? As I understand it you are going to build and publish container image, right? I think `quay.io/kubevirtci/bootstrap` would be better for that.

Reply:
Changed to `quay.io/kubevirtci/bootstrap:v20210316-d295087`. Confirmed that the image and tag exists :smile:

Reply:
Looks like you didn't push the changes? I still see the golang image in the diff

Reply:
Looks like I changed the wrong file. I'll push the correct changes. Thanks!